nodethon/README.md

```diff
@@ -0,0 +1,72 @@
+# "Nodethon" (Node.js + Python) Docker
+
+Builds and pushes a docker environment for use with tox testing. This environment contains Node.js 13 and some other JavaScript dependencies, besides a series of python versions allowing multi-version tox testing locally and in CI services.
+
+The "sq" suffix indicates that the ["sonarqube"](https://www.sonarqube.org) scanner has been included in the image. This scanner allows tox builds to upload coverage and other reports to a sonarqube instance.
```

Comment:
let's omit the sq suffix for this container. We added that to toxic because it wasn't part of the earlier offering.